test/scripts/install-cno-kind.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -ex
+
+export GO111MODULE="on"
+
+# Install kubectl
+pushd $GOPATH/src/k8s.io/kubernetes/
+sudo ln ./_output/local/go/bin/kubectl /usr/local/bin/kubectl
+sudo ln ./_output/local/go/bin/e2e.test /usr/local/bin/e2e.test
+popd
+
+# Install kind
+mkdir -p $GOPATH/bin
+wget -O $GOPATH/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-amd64
```

Comment:
with this patch we can use 0.8.1 https://github.com/openshift/cluster-network-operator/pull/663

Reply:
@aojea nice, I am testing your PR now. Only issue off the top of my head with 0.8.x is KIND switches from the default network bridge which in turn bumps the subnet up an octet. I need to go through e2e in ovn-kubernetes to make the tests network agnostic.